handlers/api.go

```diff
@@ -1326,17 +1327,39 @@ func getEpoch(epoch int64) ([]interface{}, error) {
 // ApiValidator godoc
 // @Summary Get up to 100 validators
 // @Tags Validator
+// @Description Searching for too many validators based on their pubkeys will lead to an "URI too long" error
 // @Produce  json
 // @Param  indexOrPubkey path string true "Up to 100 validator indicesOrPubkeys, comma separated"
 // @Success 200 {object} types.ApiResponse{data=[]types.APIValidatorResponse}
 // @Failure 400 {object} types.ApiResponse
+// @Router /api/v1/validator/{indexOrPubkey} [get]
+func ApiValidatorGet(w http.ResponseWriter, r *http.Request) {
+	apiValidator(w, r)
+}
+
+// ApiValidator godoc
+// @Summary Get unlimited validators
+// @Tags Validator
+// @Produce  json
+// @Param  indexOrPubkey path string true "Validator indicesOrPubkeys, comma separated"
+// @Success 200 {object} types.ApiResponse{data=[]types.APIValidatorResponse}
+// @Failure 400 {object} types.ApiResponse
 // @Router /api/v1/validator/{indexOrPubkey} [post]
-func ApiValidator(w http.ResponseWriter, r *http.Request) {
+func ApiValidatorPost(w http.ResponseWriter, r *http.Request) {
+	apiValidator(w, r)
+}
 
+func apiValidator(w http.ResponseWriter, r *http.Request) {
```

Comment:
Please leave a comment explaining this "odd at the first glance" construction.

Maybe something like this:
```suggestion
// This endpoint supports both GET and POST but requires different swagger descriptions based on the type
func apiValidator(w http.ResponseWriter, r *http.Request) {
```

Reply:
Sure, added.